client/app.jsx

```diff
@@ -21,12 +22,14 @@ class App extends React.Component {
 
   componentDidMount() {
 
+    console.log(this.state.listingId)
+
     var listingId = this.state.listingId
 
     // console.log('axios called for listingId: ', listingId)
-    axios.get(`http://localhost:8888/api/reservation/${listingId}`)
```

Comment:
I noticed you had this before and forgot to mention. Was there a reason behind putting the whole url rather than the specific end point? Im just curious, i know you changed it.